tracer/test/snapshots/NetActivitySdkTests.verified.txt

```diff
@@ -0,0 +1,456 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: Samples.NetActivitySdk.server,
+    Resource: Ctor1,
+    Service: Samples.NetActivitySdk,
+    Type: web,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: Samples.NetActivitySdk,
+      otel.status_code: STATUS_CODE_UNSET,
```

Comment:
I kind of wonder if we _should_ be adding these 🤔 It doesn't seem like they add much value? But then we won't _know_ if they will I guess...

Reply:
That's a fair thought...we could decide to add the tag only if the user explicitly configured it. I believe the Datadog Agent's OTLP ingest performed the behavior of always setting the `otel.status_code`, which inspired the existing .NET Tracer implementation